creator-node/src/utils.js

```diff
@@ -152,6 +152,13 @@ const ipfsCat = (path, req, timeout = 1000, length = null) => new Promise(async
     let options = {}
     if (length) options.length = length
     if (timeout) options.timeout = timeout
+
+    // using a js timeout because IPFS cat sometimes does not resolve the timeout and gets
+    // stuck in this function indefinitely
+    setTimeout(() => {
+      return reject(new Error('ipfsCat timed out'))
```

Comment:
will this just unblock the caller while the operation is hung indefinitely?

Reply:
yes but there's no way to force the ipfs operation to close. right now that's effectively whats happening, the operation is hung and the caller is blocked so i'm just freeing up the caller

Reply:
makes sense

Reply:
agreed, resolving